_data/hosting.yml

```diff
@@ -539,7 +539,14 @@ websites:
     twitter: WixHelp
     facebook: wix
     img: wix.png
-
+    
+  - name: World4You
+    url: https://www.world4you.com
+    img: world4you.png
+    tfa:
+      - software
```

Comment:
```suggestion
      - proprietary
```

Reply:
The doc page says:
> The app is compatible with the free Time-Based One-Time (TOTP) standard...

They even refer to Microsoft Authenticator as an alternative. So this should just be "totp".

Reply:
Nice catch @phallobst!